src/msw/toplevel.cpp

```diff
@@ -246,6 +249,34 @@ WXHWND wxTopLevelWindowMSW::MSWGetParent() const
     return (WXHWND)hwndParent;
 }
 
+bool wxTopLevelWindowMSW::HandleDPIChange(const wxSize newDPI, const wxRect newRect)
+{
+    if ( !m_perMonitorDPIaware )
```

Comment:
I wonder why do we bother checking for this: AFAICS this can't be called at all in this case as we're simply not going to get `WM_DPICHANGED`. If anything, I'd probably set `m_perMonitorDPIaware` to true here if we do get it, although this shouldn't happen neither in theory.

P.S. After reading the entire patch, I don't think `m_perMonitorDPIaware` is actually used at all... Perhaps we don't even need it (nor the function used for initializing it)?

Reply:
It is used, explained in https://github.com/wxWidgets/wxWidgets/pull/1499#issuecomment-524089850.